docs/getting-started/install-zowe-sdks.md

```diff
@@ -86,14 +100,60 @@ npm install core-for-zowe-sdk.tgz
 
 ### Python SDK *technical preview*
 
-If you install Python SDK packages from the online registry, the required dependencies are installed automatically.
+If you install Python SDK packages from the [online registry](#installing-an-sdk-from-an-online-registry), the required dependencies are installed automatically. **[should we add this sentence to the Kotlin and Java sections above?]**
```

Comment:
@adam-wolfe - please respond

Reply:
Maven and Gradle do this, but there is not really an "installation" step. The developer would just add the Zowe Client Java or Kotlin SDK to a project's build configuration file (pom.xml for Maven or build.gradle for Gradle) and then Maven/Gradle will download any dependencies for the Java/Kotlin SDK. 

With that being said, it might be worth adding some info for "installing" the Kotlin SDK. I could be missing something, but, at least when using Maven in Eclipse, I have to install the Kotlin SDK.jar to a local repository by running `mvn install:install-file -Dfile=zowe-kotlin-sdk-0.4.0.jar -DgroupId=org.zowe.sdk -DartifactId=zowe-kotlin-sdk -Dversion=0.4.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true`

@KUGDev Should we add something about importing the Kotlin SDK as a local jar in Maven/Gradle?

Reply:
@adam-wolfe yes, it will be added with the changes for IntelliJ plug-in's docs